client/src/utils/doesClassExist.js

```diff
@@ -6,7 +6,11 @@ function doesClassExist(className, stylesheets) {
         return true;
       }
     } else if (typeof stylesheet === 'string') {
-      if (stylesheet.includes(`.${className}::before`)) {
+      if (
```

Comment:
Does DOM search not work in this case ? Instead of parsing stylesheet and depending on the structure.
```
const elementsWithClass = document.querySelectorAll(`.${className}`);
return elementsWithClass.length > 0;
```